spec/javascript/components/supplemental_application/sections/__snapshots__/RentalAssistance.test.js.snap

```diff
@@ -566,8 +573,9 @@ Array [
             <label
               className="form-label"
               htmlFor="form-recipient"
+              id="label-recipient"
             >
-              Recipient 
+              Recipient
```

Comment:
also curious where these removed whitespaces come from

Reply:
not sure why whitespace would be removed for the snapshots